content/collections.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Collections and Models
+---
+
+After reading this guide, you'll know:
+
+1. What the different flavors of Mongo Collection in Meteor are, and how to use them.
```

Comment:
Yeah not sure what the "etiquette" here is. We should see what MongoDB does on their website, and do the same thing.